software/moltres/moltres.md

```diff
@@ -47,8 +55,7 @@ root of the Moltres directory.
 
 ## Development
 
```

Comment:
Can you add a note about the `CONTRIBUTING.md` file? I think it's a good reference for people to use and this makes it more consistent to changes made in the Moltres repo README.